packages/components/src/side_scroller/index.jsx

```diff
@@ -10,7 +10,8 @@ import Controls from './controls';
 import { DISABLE_SCROLL_DISTANCE, SHIFT_PERCENT, SHIFT_TOLERANCE, ANIMATION_DURATION, ANIMATION_FPS } from './constants';
 import { getAnimationPosition } from './utils';
 
-const SideScroller = ({ className: passedClassName, children, ...cleanProps }) => {
+const SideScroller = (props) => {
+  const { className: passedClassName, children, onShift, onDragStart, ...cleanProps } = props;
```

Comment:
moving the props object explode thingy down here actually makes it harder for IDEs to analyse props. please move it back into line 13